src/observer.ts

```diff
@@ -190,8 +203,12 @@ export class DataLayerObserver {
    * adding the operator, the DataHandler will be removed to prevent unexpected data processing.
    * @param handler to add operators to
    * @param options for operators used to configure each Operator
+   * @param destination The javascript function to execute (must be one of destination or fsApi)
+   * @param fsApi The special FullStory constant to be executed (must be one of destination or fsApi)
    */
-  private addOperators(handler: DataHandler, options: OperatorOptions[], destination: string | Function) {
+  private addOperators(handler: DataHandler, options: OperatorOptions[],
+    destination: string | Function | undefined = undefined, fsApi: string | undefined = undefined,
```

Comment:
`destination` and `fsApi` can be declared optional with `destination?` and `fsApi?`. Is there a need to add the type `undefined` and assign it `undefined`?  That would be automatic with the optional.